regulations/tasks.py

```diff
@@ -59,4 +76,4 @@ def fetch_file(path, key, name):
     s3 = session.client('s3')
     dest = os.path.join(path, name)
     s3.download_file(settings.BUCKET, key, dest)
-    return dest
+    return open(dest, "rb")
```

Comment:
It feels a little odd to return a file object here, but I do think it helps in overall readabiity.